backend.rb

```diff
@@ -39,12 +40,19 @@ def build_url(path)
     "#{ENV["AUTHY_API_URL"]}#{path}"
   end
 
-  def build_params_for_authy(authy_id)
-    {
-      api_key: ENV["AUTHY_API_KEY"],
-      authy_id: authy_id
-    }
+  def build_registration_token(authy_id, app_id, hmac_secret, account_sid)
+
+      payload = {:jti => '#{app_id}-#{Time.now.to_i}', :iss =>  app_id, :sub => account_sid, :nbf => Time.now.to_i - 60, :exp => Time.now.to_i + 10*3600,
+        :grants => {
```

Comment:
We should contribute to the twilio helper libraries also, for this token to be provided through a builder like other JWTs.